src/api/analytics.ts

```diff
@@ -107,7 +133,7 @@ export class Analytics {
     }: GetEnrollmentsQueryOptions): D2ApiResponse<AnalyticsResponse> {
         return this.d2Api.get<AnalyticsResponse>(
             `/analytics/enrollments/query/${programId}`,
-            options as AnalyticsOptions
+            options as Omit<GetEnrollmentsQueryOptions, "programId">
```

Comment:
I was a bit confused as to why we needed to cast the options object, it seemed compatible with the type. Apparently, it comes from the early unpacking that extracts programId. So let's keep it simple: `options: GetEnrollmentsQueryOptions` and now the cast can be removed.